src/main/java/io/anserini/kg/freebase/FreebaseNode.java

```diff
@@ -68,10 +68,13 @@ public String uri() {
 
   public static String cleanUri(String uri) {
     if (uri.charAt(0) == '<') {
-      return uri.substring(1, uri.length() - 1).toLowerCase();
-    } else {
-      return uri;
+      uri = uri.substring(1, uri.length() - 1).toLowerCase();
     }
+
+    uri = uri.replaceAll("^http://rdf.freebase.com/ns/", "fb:");
```

Comment:
Minor - we can do `fb:` and `fbkey:` as static final strings instead of hard-coding them here?